OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -48,8 +48,9 @@ public sealed class WorldRenderer : IDisposable
 		internal WorldRenderer(ModData modData, World world)
 		{
 			World = world;
-			TileSize = World.Map.Grid.TileSize;
-			TileScale = World.Map.Grid.Type == MapGridType.RectangularIsometric ? 1448 : 1024;
+			var map = World.Map;
+			TileSize = map.Grid.TileSize;
+			TileScale = map.Grid.Type == MapGridType.RectangularIsometric ? 1448 : 1024;
```

Comment:
suggestion to then also hide this in one of the IMap interfaces. seems wrong to hardcode these constants in world renderer.

Reply:
These are actually within `IMap` now, as the WorldRenderer now uses an `IMap` for its map variable.